job-engine/jbatch/src/main/java/org/eclipse/kapua/job/engine/jbatch/JobEngineServiceJbatch.java

```diff
@@ -189,4 +189,9 @@ public void stopJob(KapuaId scopeId, KapuaId jobId) throws KapuaException {
             throw new JobStopppingException(e, scopeId, jobId);
         }
     }
+
+    @Override
+    public void cleanJobData(KapuaId scopeId, KapuaId jobId) throws KapuaException {
```

Comment:
Add:

Argument validation
Auth check (JOB_DOMAIN, Action.delete, null)
Check job existence
Check job not running
Throw a `CleanJobDataException` (see JobStoppingException) if `JbatchDriver.cleanJobData(scopeId, jobId)` fails.